src/circuit/gadget/add_chip.rs

```diff
@@ -1,3 +1,5 @@
+//! `Add` chip implemetation.
```

Comment:
_From str4d ([r2550151848](https://github.com/zcash/orchard/pull/471#discussion_r2550151848))_

```suggestion
//! `Add` chip implementation.
```

Reply:
FIXED